ClassTypes/PowerShell/EthicalHacking/README.md

```diff
@@ -1,40 +1,71 @@
 # Introduction
-These scripts will guide you to create and setup an Azure Lab Services lab account that is configured to run an [ethical hacking class](https://docs.microsoft.com/en-us/azure/lab-services/classroom-labs/class-type-ethical-hacking). Part 1 of these instructions will be to create the lab account on Azure. Part 2 of these instructions will be to prepare the template VM instance of the newly created lab to be used by your class.
+
+These scripts will guide you to create and setup an Azure Lab Services lab that is configured to run an [ethical hacking class](https://docs.microsoft.com/azure/lab-services/classroom-labs/class-type-ethical-hacking). Part 1 of these instructions will be to create the lab plan on Azure. Part 2 of these instructions will be to prepare the template VM instance of the newly created lab to be used by your class.
 
 - - - -
 
-## Part 1 - Create a Lab Account 
-This script will help create a Lab Account on your Azure account from your local machine.
+## Part 1 - Create a lab plan
+
+This script will help create a lab plan in your Azure subscription from your local machine.
+
+### Prerequisites
 
-> NOTE: Script must be executed using administrator privileges.
+To create a lab using the following instructions, you must have
+
+- Contributor permissions on the subscription in which the lab will be created
```

Comment:
Should we enable users to run the script with only resource-group level permissions?  (have we ever gotten that request?)  It would be possible if they passed in an RG instead of created it on the fly in the script, but I don't know if it's needed.

Reply:
The Create-EthicalHackingLabplanAndLab.ps1 was meant as a quick way to create a demo.  I've never had a request to have an RG or lab plan passed into the script.  As far as I know, people just skip straight to the updating the template script if they are creating the lab for real.